widgetssdk/src/main/java/com/glia/widgets/chat/model/ChatItems.kt

```diff
@@ -0,0 +1,248 @@
+package com.glia.widgets.chat.model
+
+import android.content.Context
+import com.glia.androidsdk.chat.AttachmentFile
+import com.glia.androidsdk.chat.ChatMessage
+import com.glia.androidsdk.chat.SingleChoiceOption
+import com.glia.widgets.chat.adapter.ChatAdapter
+import com.glia.widgets.helper.isDownloaded
+
+internal abstract class ChatItem(@ChatAdapter.Type val viewType: Int) {
+    abstract val id: String
+    abstract val timestamp: Long
+
+    open fun areContentsTheSame(newItem: ChatItem): Boolean = this == newItem
+}
+
+/**
+ * this is the same as [ChatItem], but should be used with non-local chat items
```

Comment:
Minor suggestion. Lets start the comment from a capital letter, maybe?
> This is the same as ...

Reply:
fixed - [25300dc](https://github.com/salemove/android-sdk-widgets/pull/679/commits/25300dc0374c728a1e2336716a6be423e0717691)